output.go

```diff
@@ -42,6 +42,31 @@ func FormatYaml(writer io.Writer, value interface{}) error {
 	return nil
 }
 
+// FormatYaml writes out value as yaml to the writer, unless value is nil.
+func FormatErrYaml(writer io.Writer, value interface{}) error {
+	if value == nil {
+		return nil
+	}
+	msg := value.(error).Error()
```

Comment:
You can't guarantee this, you should add a check for it. It would also eliminate the nil check above.

i.e. doing it in a switch.

Reply:
Usually don't but in the above example I actually do, because this func is only called when `err !- nil`
```
err != nil && !IsErrSilent(err) 
```

Reply:
The we should make the type of the function more stricter, that way we get type safety.

Reply:
Yeah true :+1: